lib/modules/onc_program/test/onc_openid_connect_test.rb

```diff
@@ -587,6 +587,26 @@ def create_signed_token(payload: @payload, key_pair: @key_pair, kid: @jwk.kid)
       assert_match 'Bad response code: expected 200, 201, but found 404. ', exception.message
     end
 
+    it 'fails if fetching the user is rejected and the user is Patient, Practitioner or RelatedPerson resource' do
+      stub_request(:get, @decoded_payload['fhirUser'])
+        .to_return(status: 401)
+
+      exception = assert_raises(Inferno::AssertionException) { @sequence.run_test(@test) }
+
+      assert_match 'Bad response code: expected 200, 201, but found 401. ', exception.message
+    end
+
+    it 'passes if fetching the user is rejected and the user is Person resource' do
```

Comment:
So we are completely disallowing Person, even though SMART allows it?  I understand that in most cases it is NOT valid to use Person, but we cannot risk failing systems that use it properly, can we?